.github/workflows/Documenter.yml

```
@@ -10,14 +10,13 @@ jobs:
     name: Documentation
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v5
       - uses: julia-actions/setup-julia@v2
         with:
-          version: '1.10'
+          version: '1'
```

locking it to a precise minor version seems good.
Because it stops doctests breaking if julia decides to change the string representation of things again

```suggestion
          version: '1.10'
```